app/controllers/invoice_lists_controller.rb

```diff
@@ -134,6 +135,14 @@ def assign_attributes # rubocop:disable Metrics/AbcSize
     end
     entry.creator = current_user
     entry.invoice = parent.invoices.build(permitted_params[:invoice])
+
+    if params[:invoice_items].present?
+      entry.invoice.invoice_items = params[:invoice_items].map do |type|
```

Comment:
ich finde es nicht sehr elegant den type als parameter in der url mitzugeben. 
die url wird sehr lange und cryptisch. gibt es eine andere möglichkeit?